src/client/components/Button/ButtonComponent.styles.css

```diff
@@ -0,0 +1,35 @@
+.button {
+  font-family: 'Nunito Sans', 'Helvetica Neue', Helvetica, Arial, sans-serif;
+  font-weight: 700;
+  border: 0;
+  border-radius: 3em;
+  cursor: pointer;
+  display: inline-block;
+  line-height: 1;
+}
+
+.button--primary {
```

Comment:
These colors are not part of the mockup. It  is good to have a primary and secondary classes to standarize the styles of the primary buttons and the ones for the secondary buttons. You could specify the correct color. Fx, primary buttons with #8E0EF2 and secondary buttons with #929B90. But then the classes would need to be passed as parameters when using the button.

Reply:
i will make one class ... and will remove primary and secondary classes